storybooknative/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@kiwicom/orbit-design-tokens": "^0.2.1",
+    "lodash.omit": "^4.5.0",
```

Comment:
With the existing solution in https://github.com/kiwicom/mobile/blob/master/packages/shared/src/PlatformStyleSheet.js, it probably won't be necessary to add this library.

Reply:
I took inspiration from the existing solution but, I decided to a different approach. In the existing solution, we iterate through every style's property.

I mine solution, I iterate through platform types and check if styles for particular platform exists. I suppose that is more effective in many cases then go through all property in style object.

Also, the difference between what we use on **mobile** and what we need for the **RNW Orbit** is in requirements.
Because we expected in the **RNW Orbit** that we can send `Object` and `Function` as the parameter to the `styled` function.

If we don't want to use omit from the `lodash`, I can write own omit function.  But I would like to know where is the problem with this approach?

possible custom omit function implementation.
```
const omit = (object, prop) => {
  const objectWithoutProp = Object.assign({}, object);
  delete objectWithoutProp[prop];
  return objectWithoutProp;
};
```

Reply:
I definitely don't think we should make our own implementation of what already exists. The best code is the one you don't have to maintain 😄 

I do agree here that could be more efficient not to copy all properties always yes. 

Then there is the never ending discussion lodash vs ramdba 🤷‍♂️

Reply:
I like the ramda also. 
But on the end, I am going it rewrite it to the how @RobinCsl suggest. 😉
https://github.com/kiwicom/orbit-components/pull/550#discussion_r229665315